pennylane/devices/qubit/adjoint_jacobian.py

```diff
@@ -77,7 +65,7 @@ def adjoint_jacobian(tape: QuantumTape):
         wire_map = {w: i for i, w in enumerate(tape.wires)}
         tape = qml.map_wires(tape, wire_map)
 
-    ket = _get_output_ket(tape)
+    ket = state if state is not None else get_final_state(tape)[0]
```

Comment:
```suggestion
    ket = state or get_final_state(tape)[0]
```

Reply:
NumPy is complaining that the truth value of `state` is ambiguous, so I think this is necessary.

Reply:
well that's annoying. sorry about the suggestion.